comm/src/main/scala/coop/rchain/catscontrib/Capture.scala

```diff
@@ -0,0 +1,36 @@
+package coop.rchain.catscontrib
```

Comment:
potentially we could start another top-level project for this.  i have some code that go there as well...

Reply:
perhaps `library` so it's more general than cats extension stuff?

Reply:
reading my mind https://github.com/rchain/rchain/pull/279#issuecomment-366083133 :)